inst/www/shared/shiny.js

```diff
@@ -3296,6 +3299,14 @@
         sendOutputHiddenStateDebouncer.immediateCall();
     });
 
+    function filterEventsByNamespace(namespace, handler) {
+      return function(e) {
+        if (e.namespace.indexOf(namespace) === 0) {
```

Comment:
I think this comparison isn't quite right. It happens to work for the `bs` events, but I don't think it will work in general because the components of the namespace can be re-ordered. From the jQuery [event docs](http://learn.jquery.com/events/event-extensions/):

> When multiple namespaces are given, they are separated by periods and sorted in ascending alphabetical order.